src/main/java/org/javarosa/core/util/GeoUtils.java

```diff
@@ -98,6 +99,16 @@ public static double calculateDistance(List<LatLong> points) {
         return totalDistance;
     }
 
+    public static String geojsonToOdkGeometry(JsonNode geojsonGeometry) throws IOException {
```

Comment:
I would have preferred a String parameter but I couldn't quickly find a way to do that.